src/dto/patching-time-slot.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+import { TimeSlotDTO } from './time-slot.dto';
+
+export class PatchingTimeSlotDTO extends TimeSlotDTO {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  groupdId: string;
```

Comment:
I think you should not have `groupId`, but you should have `startDate` and `endDate`.